content/W2017/W17-event-techtalk-with-Monica-Beckwith.md

```diff
@@ -0,0 +1,18 @@
++Title: Techtalk with Monica Beckwith
```

Comment:
it's not showing up on the test site because of these `+`s at the start of your lines